### ISSUE TITLE
Allow Scripts to Be Run Anywhere in Project

### ISSUE THREAD

**Issue by joelruns** [sdispater/poetry#448]
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.

## Issue

Would it make sense to also append `sys.path.append(directory_of_pyproject-toml);` to `cmd` variable like on line 32 of `poetry/console/commands/run.py`?

This would allow `poetry run script_from_pyproject-toml` to be runnable from anywhere in the project without producing errors like this:
```
File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'mymodule'
```
Of course we can just append `PYTHONPATH` to include the project root, but I feel that might be a bit contradictory to the philosophy of `poetry` removing an otherwise purely boilerplate-esque step similar to not needing to activate virtual environment. As a point of comparison, `npm` scripts can be run anywhere in the project. Unless I'm doing using the `scripts` section entirely wrong, I think this might be a reasonable fast enhancement. Thanks!

**Comment by jacebrowning** [comment]
Try this again after running `poetry develop`.

https://github.com/sdispater/poetry/issues/181 proposes this becomes the default `poetry install` behavior.

**Comment by joelruns** [comment]
Huh... this is quite nice... thank you. I'm not sure if I'm going blind but I can't seem to find this command anywhere on the documentation.

**Comment by jacebrowning** [comment]
Yeah... it's not listed here: https://poetry.eustace.io/docs/cli/

**Comment by joelruns** [comment]
Closing since the concerns are addressed in #181. Hopefully will get resolved soon!

`poetry develop` solves the problem as well.
